x-pack/plugins/gis/public/components/map/mb/view.js

```diff
@@ -64,7 +64,6 @@ export class MBMapContainer extends React.Component {
     const initialZoom = this.props.mapState.zoom;
     const initialCenter = this.props.mapState.center;
     this._mbMap = await createMbMapInstance(this.refs.mapContainer, initialZoom, initialCenter);
-    window._mbMap = this._mbMap;
```

Comment:
Not sure why `_mbMap ` was every exposed on window but this should not be so I removed it